app/src/main/java/com/framgia/wsm/screen/confirmrequestleave/ConfirmRequestLeaveViewModel.java

```diff
@@ -121,6 +119,26 @@ public void onCreateFormFormRequestLeaveError(BaseException exception) {
     }
 
     @Override
+    public void onEditFormRequestLeaveSuccess() {
+        mNavigator.finishActivityWithResult(Activity.RESULT_OK);
+    }
+
+    @Override
+    public void onEditFormFormRequestLeaveError(BaseException exception) {
+        Log.e(TAG, "ConfirmRequestLeaveViewModel", exception);
+    }
+
+    @Override
+    public void onDeleteFormRequestLeaveSuccess() {
+        mNavigator.finishActivityWithResult(Activity.RESULT_OK);
+    }
+
+    @Override
+    public void onDeleteFormFormRequestLeaveError(BaseException exception) {
+        Log.e(TAG, "ConfirmRequestLeaveViewModel", exception);
```

Comment:
ERROR: [LongLogTag] The logging tag can be at most 23 characters, was 28 (ConfirmRequestLeaveViewModel)
 Log tags are only allowed to be at most 23 tag characters long.